eShopsolution.Data/Scripts/Product.sql

```diff
@@ -8,5 +8,6 @@ CREATE TABLE [dbo].[Product](
     [Stock] [int] NULL,
     [ViewCount] [int] NULL,
     [DateCreated] [datetime] NULL,
-    [IsFeatured] [bit] NULL
+    [IsFeatured] [bit] NULL,
```

Comment:
Chỗ này sẽ không chạy được khi table đã có data, nên hôm bữa a2 kiêu viết script kiểu nếu column đó chưa có thì thêm vào
IF COL_LENGTH('Product','Description') IS NULL
BEGIN
	ALTER TABLE [dbo].ProductADD  Description [nvarchar](100) NULL
END

Reply:
https://www.w3schools.com/sql/sql_alter.asp

Reply:
thêm thẳng zo db luôn r a2

Reply:
thẳng vô db thì xóa column đó ra, tạo lại script, cho giống đi làm